zilencer/management/commands/populate_db.py

```diff
@@ -209,6 +209,16 @@ def handle(self, **options):
             zulip_realm_bots.extend(all_realm_bots)
             create_users(realms, zulip_realm_bots, bot_type=UserProfile.DEFAULT_BOT)
 
+            zulip_default_bots = [
+                ("Zulip Default Bot", "default-bot@zulip.com"),
+            ]
+            create_users(realms, zulip_default_bots, bot_type=UserProfile.INCOMING_WEBHOOK_BOT)
+
+            zulip_webhook_bots = [
+                ("Zulip Webhook Bot", "webhook-bot@zulip.com"),
+            ]
+            create_users(realms, zulip_webhook_bots, bot_type=UserProfile.INCOMING_WEBHOOK_BOT)
+
```

Comment:
It looks like the webhook-bot chunk isn't in the commit whose description claims to add it.  I think to fix this, you want to do the following rebase:

(1) Move commit 3 ("Add webhook-bot during populate_db" to be the second commit
(2) Move the hunk here that creates webhook-bot from the "Add is_webhook option" commit to the "Add webhook-bot commit".